[Enhancement] Optimize log print and error message.

luohaha opened the pull request:
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：
Fixes #11751

## Problem Summary(Required) ：
In this pr, there are three part improvements:
1. setup error msg in agent task return msg.
2. show backends state info, when fe can't find useful backends.
3. remove useless error msg when help zip file not found

E.G, When frontend can't find useful backends, the error msg now can show the current state of backends, and you will find out why they can't be used.
```
mysql> select * from tbl;
ERROR 1064 (HY000): Backend not found. Check if any backend is down or not. backend: [11001-192.168.58.1-9660 alive: false inBlacklist: false] [11097-192.168.58.1-9661 alive: false inBlacklist: false] [11098-192.168.58.1-9662 alive: false inBlacklist: false]
```
## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/StarRocks/starrocks?pullRequest=11635) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/StarRocks/starrocks?pullRequest=11635) before we can accept your contribution.<br/><hr/>**luoyixin** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/StarRocks/starrocks?pullRequest=11635) it.</sub>

wanpengfei-git commented:
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 23 (30.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

wanpengfei-git commented:
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 23 (30.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

wanpengfei-git commented:
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 23 (30.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

luohaha committed:
[Enhancement] Optimize log print and error message.

In this pr, there are three part improvements:
1. setup error msg in agent task return msg.
2. show backends state info, when fe can't find useful backends.
3. remove useless error msg when help zip file not found

wanpengfei-git commented:
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 23 (30.43%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

wanpengfei-git commented:
run starrocks_admit_test

chaoyli commented on fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java:
```diff
@@ -1619,6 +1619,28 @@ private TNetworkAddress toBrpcHost(TNetworkAddress host) throws Exception {
+            for (Map.Entry<Long, Backend> entry : this.idToBackend.entrySet()) {
+                Long backendID = entry.getKey();
+                Backend backend = entry.getValue();
+                infoStr += String.format("[%d-%s-%d alive: %b inBlacklist: %b] ", backendID, backend.getHost(), 
```
IP is enough. The backendID and port is not necessary

chaoyli replied:
```
backend: xxx. error: xxx
```
is clear

luohaha replied:
done

wanpengfei-git commented:
run starrocks_admit_test

wanpengfei-git commented:
run starrocks_admit_test

wanpengfei-git commented:
#### [FE PR Coverage Check]

:disappointed: **fail** : 7 / 24 (29.17%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/HelpModule.java|0|1|00.00%|[264]|
|:large_blue_circle:|com/starrocks/sql/ast/RangePartitionDesc.java|0|1|00.00%|[134]|
|:large_blue_circle:|com/starrocks/qe/Coordinator.java|7|22|31.82%|[1622, 1623, 1627, 1633, 1634, 1635, 1636, 1637, 1638, 1639, 1640, 1641, 1643, 1683, 1880]|

luohaha commented:
https://github.com/Mergifyio backport branch-2.5
